SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialDispatcher.cpp

```diff
@@ -102,14 +78,6 @@ void SpatialDispatcher::ProcessOps(const TArray<Worker_Op>& Ops)
 				SpatialWorkerFlags->SetWorkerFlag(UTF8_TO_TCHAR(Op.op.flag_update.name), UTF8_TO_TCHAR(Op.op.flag_update.value));
 			}
 			break;
-		case WORKER_OP_TYPE_METRICS:
```

Comment:
Was this intentionally removed? Doesn't look like it's called anymore.

Reply:
It's called here: https://github.com/spatialos/UnrealGDK/pull/2941/files/034a7c457238b899031b62f928e3970b54537639#diff-68846c07228a3db21f40ca2e7ec562724289cb4c4d2d9f622123b24e0b98db39R24-R27